### ISSUE TITLE
200 x 200 vieport is too small for some diagrams

### ISSUE THREAD

**Issue by RomanHotsiy** [ChappIO/gatsby-remark-mermaid#3]
E.g. one of the example diagrams from https://mermaidjs.github.io/ (see bellow) is broken when rendered with `200x200` but is ok when rendering with e.g. `800x800`.

Other diagrams seem to be unchanged from this change.

```
gantt
        dateFormat  YYYY-MM-DD
        title Adding GANTT diagram functionality to mermaid
        section A section
        Completed task            :done,    des1, 2014-01-06,2014-01-08
        Active task               :active,  des2, 2014-01-09, 3d
        Future task               :         des3, after des2, 5d
        Future task2               :         des4, after des3, 5d
        section Critical tasks
        Completed task in the critical line :crit, done, 2014-01-06,24h
        Implement parser and jison          :crit, done, after des1, 2d
        Create tests for parser             :crit, active, 3d
        Future task in critical line        :crit, 5d
        Create tests for renderer           :2d
        Add to mermaid                      :1d
```
Actual when `200x200`:

![image](https://user-images.githubusercontent.com/3975738/61365158-e2c71b80-a88f-11e9-99e7-dd35d0ca0482.png)

Expected when `800x800`:

![image](https://user-images.githubusercontent.com/3975738/61365010-9bd92600-a88f-11e9-9be2-4bf0828a121f.png)

I can send a PR but it's just a few lines so I think you can do it faster ⚡️ 

May be related to #1

**Comment by ChappIO** [comment]
Thanks for the request. I'll most likely implement #1 this weekend so you can set your own viewport size.
Expect a release this weekend.

**Comment by ChappIO** [comment]
@RomanHotsiy I just released version 1.1.0.

You can now configure the viewport size in the plugin settings. Let me know if that does the trick for you.

**Comment by RomanHotsiy** [comment]
Thanks! But I would anyway increase defaults. Gannt diagrams are all broken with width 200

P.S. I ended up just copying your code inside the project as I had issues with puppeteer in docker. I will open a separate issue.
